### DISCUSSION TITLE
Detached Mode

### DISCUSSION THREAD

**Discussion by abbottcam** [SmartHome-yourself/sonoff-tx-ultimate-for-esphome#57] [Q&A]
Hi there, Ive just got myself a 3 gang TX Ultimate, Im rather new. Im looking to detach my relays, and instead control Zigbee2Mqtt scenes. Im sure this is possible, but would anyone be able to point me in the right direction?

thanks

James

**Comment by robbiedehand** [comment]
![image](https://github.com/user-attachments/assets/4bb02d28-dbd9-4406-b8b3-5afaf8054731)

**Reply by robbiedehand** [reply]
Just adjust as shown in esp and make automation in ha ,touchfield number is pressed than switch light (or what you want)
